src/README.md

```diff
@@ -30,12 +22,32 @@ have read it, and we will take care to occasionally give a refresher on the
 basics where appropriate. You can skip straight to this book if you want;
 just know that we won't be explaining everything from the ground up.
 
-We're going to dig into exception-safety, pointer aliasing, memory models,
-compiler and hardware implementation details, and even some type-theory.
-Much text will be devoted to exotic corner cases that no one *should* ever have
-to care about, but suddenly become important because we wrote `unsafe`.
-
-We will also be spending a lot of time talking about the different kinds of
-safety and guarantees that programs could care about.
-
-[trpl]: ../book/index.html
+This book exists primarily as a high-level companion to [The Reference][ref].
+Where The Reference is essentially a giant exhaustive list of every individual
+component of Rust and what semantics it has, The Rustonomicon serves to describe
+how to put those pieces together, and the issues that you will have in doing so.
+
+The Rustonomicon is the "integration tests" to The Reference's "unit tests".
```

Comment:
```suggestion
```
Not sure this analogy works... The reference is aimed at being *formal* but should cover everything... tests are not that...